torch_xla/csrc/ops/linear_interpolation.cpp

```diff
@@ -2,15 +2,14 @@
 
 #include "torch_xla/csrc/helpers.h"
 #include "torch_xla/csrc/lowering_context.h"
-#include "torch_xla/csrc/ops/xla_ops.h"
 
 namespace torch_xla {
 namespace ir {
 namespace ops {
 
 LinearInterpolation::LinearInterpolation(const Value& value,
                                          const Value& new_value, double alpha)
-    : Node(xla_moving_average, {value, new_value}, value.xla_shape(),
```

Comment:
I think we should leave the opkind here as it is . `LinearInterpolation` seems to be doing very similar thing as lerp but we have a lerp op implemented separted. Having 2 node with same op kind is confusing. Maybe one day we can refactor the `LinearInterpolation` node completely but until then it is probably better to leave it as it is.